### PR TITLE
Profile view + navbar descriptive

### PR DESCRIPTION
**Pull request by raphaeldieu**
navbar moved to descriptive
user can see his profile when logged in

### CODE REVIEW

File: app/assets/javascripts/angular/controllers/users/myprofile.js.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  
+  $scope.first_name = user_loggedin.first_name;
+  $scope.last_name = user_loggedin.last_name;
+  $scope.email = user_loggedin.email;
+  $scope.phone = user_loggedin.phone;
+  $scope.age = user_loggedin.age;
+  $scope.occupation = user_loggedin.occupation;
+  $scope.gender = user_loggedin.gender;
+  
+  if(user_loggedin.is_smoker){
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/users/myprofile.js.erb]: can be written as `$scope.is_smoker = user_loggedin.is_smoker ? "Yes" : "No";`

File: app/assets/javascripts/angular/controllers/users/myprofile.js.erb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+app.controller('MyProfileCtrl', ['$scope', '$http', '$auth', '$location', function($scope, $http, $auth, $location){
+    
+  $scope.authResponse = $auth.validateUser()
+  var user_loggedin = $scope.authResponse.$$state.value
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/users/myprofile.js.erb]: Should use camelCase in javascript

**Comment by raphaeldieu** [comment]
updated

**Comment by raphaeldieu** [comment]
camel case updated

**Comment by h8rry** [comment]
The rest looks good.

**Pull request merged**